cmd/version.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+
+	goVersion "github.com/christopherhein/go-version"
+	"github.com/spf13/cobra"
+)
+
+var (
+	shortened = false
```

Comment:
- Default value is already false, this should be rewritten as `shortened bool`
- Lets move this declaration down into `func versionCmd()`. The others need to be package level because we're setting them at build time with linker magic, but this is a flag so it shouldn't be declared at the package level.